src/arrayMethodReduce.js

```diff
@@ -5,7 +5,19 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let accumulator = initialValue;
+    let startValue = 0;
+
+    if ([...arguments].length < 2) {
```

Comment:
You can change it to arguments.length without spread operator

Reply:
done 👌
oh, yes, exactly! for some reason I like to spread everything))))